src/Storefront/Resources/views/storefront/layout/meta.html.twig

```diff
@@ -5,6 +5,7 @@
     {% set metaDescription = metaInformation.metaDescription|striptags|trim|u.truncate(maxLength ?? 255, '…') %}
     {% set metaTitle = metaInformation.metaTitle|striptags|trim %}
     {% set metaKeywords = metaInformation.metaKeywords|striptags|trim %}
+    {% set metaAuthor = (metaInformation.author != '') ? metaInformation.author : config('core.basicInformation.metaAuthor') %}
```

Comment:
Where does `metaInformation.author` come from then? Can this ever be set to something after this change? Or only via custom plugin/pageloader implementation?

Reply:
Okay but when this is the case the setting from config will be ignored and this is also intended behaviour? Or should it be the other way around? As soon as I use the admin setting it will be used globally.

Reply:
There is no way to set the author via admin under SEO via category or landingpages.

But plugins could do something like that for specific pages:
```php
$page->getMetaInformation()?->setAuthor('Great Author from some Plugin');
```

In that case, the value from the plugin would be used first.